Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Scene.h

```diff
@@ -159,10 +164,9 @@ namespace AZ
 
             bool HasOutputForPipelineState(RHI::DrawListTag drawListTag) const;
 
-            AZ::RPI::CullingScene* GetCullingScene()
-            {
-                return m_cullingScene;
-            }
+            AzFramework::IVisibilityScene* GetVisibilityScene() const { return m_visibilityScene; }
+
+            AZ::RPI::CullingScene* GetCullingScene() const { return m_cullingScene; }
```

Comment:
I know sometimes it isn't possible, but would be good if the constness of the function matched the constness of the return type.

Reply:
yeah, in this case it can't be const - RPI::Scene owns the Culling Scene because it makes sense for it to be the owner (1:1 relationship) but people asking for the CullingScene will often want to update it somehow.